src/adapters/EOFeedAdapter.sol

```diff
@@ -177,6 +183,14 @@ contract EOFeedAdapter is IEOFeedAdapter, Initializable {
         return priceData.eoracleBlockNumber;
     }
 
+    function _normalizePrice(uint256 price) internal view returns (int256) {
+        if (_inputDecimals >= _outputDecimals) {
```

Comment:
to save cost if _inputDecimals == _outputDecimals returns the price

Reply:
no need with the new optimization. an 'if' is more expensive than dividing by 1.